misc/community-marketing-kit.md

```diff
@@ -0,0 +1,71 @@
+# Guide to promoting Firefox in your language: a community marketing kit
+
+For a community new or well established, it is always exciting to reach the milestone when the localized Firefox is ready for the first or a major release.  It takes months of translation, testing and QA to get to the finish line. However, the work is not done.  Now it comes to the challenging part of promoting the product in your language and spreading the word of mouth.  In anticipation for the new Firefox launch in the autumn, this guide is intended to address some of the most frequently questions you asked.
+
+## Messages
+
+### Source:
+
+* mozilla.org! Localize it!! The site is having a makeover leading up to the autumn launch of the new Firefox.  In addition to the new look and feel, there is a lot of new content, and tone is less formal and more fun. Translate these new pages, you will have all your key messages already.
+* [Snippets](https://github.com/mozilla-l10n/engagement-l10n/tree/master/en-US/snippets/2017) and [emails](https://github.com/mozilla-l10n/engagement-l10n/tree/master/en-US/emails/2017): Only eight locales are supported on a regular basis. The team is not ready to expand to more locales due to manual process to launch them.  You can use leverage the content you see fit and publish them on your community social channels. We are in discussion with the Snippets team to support occasional request for locales not regularly supported. Check [Localizing and Testing Snippets] doc for details.
```

Comment:
`You can use leverage the content you see fit` There’s either a missing word, or some need to get removed :)